spec/features/mediation_table_spec.rb

```diff
@@ -197,6 +200,8 @@
         within('tbody td table tbody') do
           within(all('tr').last) do
             click_button('Approve')
+            expect(page).to have_button('Approve')
+            expect(page).to_not have_button('Approve', disabled: true)
```

Comment:
One thought is we could find the button in the page (using `find` or `all` with a css selector) and then make assertions on the `disabled` attribute itself.  I have used this strategy when testing for `disabled` elements in the past as it can be kind of a PITA.

I would say only if you think it's worth it to be more explicit about what we're testing.  Otherwise, perhaps a brief comment in here explaining what this is actually asserting.